Add results to `National Dutch Championship 2025` (#13824)
ppy/osu-wiki: wiki/Tournaments/NDC/2025/en.md

## Prizes

> The prizepool totals €750 and will be divided as follows:

Reviewer: Best to use past tenses as the tournament is over.
```suggestion
The prize pool totals €750 and was divided as follows:
```

| Mappool selector | ::{ flag=NL }:: [Sroj](https://osu.ppy.sh/users/4783389), ::{ flag=SG }:: [bailie](https://osu.ppy.sh/users/7537133), ::{ flag=BE }:: [Senndwich](https://osu.ppy.sh/users/12089512), ::{ flag=BE }:: [megafasator](https://osu.ppy.sh/users/8456924) |
| Mapper | ::{ flag=PL }:: [-NeBu-](https://osu.ppy.sh/users/4099626), ::{ flag=DE }:: [Mir](https://osu.ppy.sh/users/8688812), ::{ flag=NL }:: [Fubu](https://osu.ppy.sh/users/12719649), ::{ flag=NL }:: [OldEclipse](https://osu.ppy.sh/users/5174602), ::{ flag=NL }:: [Lilily](https://osu.ppy.sh/users/6502403), ::{ flag=NL }:: [Damnjelly](https://osu.ppy.sh/users/1666355), ::{ flag=NL }:: [n0ah](https://osu.ppy.sh/users/3086393), ::{ flag=US }:: [Wispy](https://osu.ppy.sh/users/11106929), ::{ flag=NL }:: [taku](https://osu.ppy.sh/users/684433), ::{ flag=NO }:: [BarkingMadDog](https://osu.ppy.sh/users/3475189), ::{ flag=NL }:: [Mijn Aim Zuigt](https://osu.ppy.sh/users/4587487), ::{ flag=DE }:: [Kojio](https://osu.ppy.sh/users/2054596) |

> | Playtester | ::{ flag=NL }:: [SecretlyShiro](https://osu.ppy.sh/users/7498203), ::{ flag=BE }:: [hexi](https://osu.ppy.sh/users/10760701), ::{ flag=BE }:: [Hanori](https://osu.ppy.sh/users/7078544), ::{ flag=NL }:: [Quinten](https://osu.ppy.sh/users/5101306), ::{ flag=FI }:: [mxu](https://osu.ppy.sh/users/18687807), ::{ flag=NL }:: [Lilily](https://osu.ppy.sh/users/6502403), ::{ flag=HK }:: [misha awa](https://osu.ppy.sh/users/14503423), ::{ flag=PT }:: [Kuronora](https://osu.ppy.sh/users/2742030), ::{ flag=NL }:: [faze-dank](https://osu.ppy.sh/users/13050074), ::{ flag=KR }:: [Amamya Kokoro](https://osu.ppy.sh/users/2511839), ::{ flag=NL }:: [Wittepoes](https://osu.ppy.sh/users/12984931), ::{ flag=NL }:: [UC2](https://osu.ppy.sh/users/6989615), ::{ flag=NL }:: [nik](https://osu.ppy.sh/users/10077264), ::{ flag=NL }:: [Alphan](https://osu.ppy.sh/users/13298387), ::{ flag=NL }:: [sofuraabu](https://osu.ppy.sh/users/7639453), ::{ flag=CN }:: [Crystal](https://osu.ppy.sh/users/1646397), ::{ flag=NO }:: [HUNDUR](https://osu.ppy.sh/users/3145033), ::{ flag=NL }:: [niqht](https://osu.ppy.sh/users/14390731), ::{ flag=DE }:: [yary](https://osu.ppy.sh/users/13300203), ::{ flag=NO }:: [Melvr](https://osu.ppy.sh/users/9211924), ::{ flag=NL }:: [Kushper](https://osu.ppy.sh/users/4832514), ::{ flag=ES }:: [A L E P H](https://osu.ppy.sh/users/6735738), ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050), ::{ flag=RO }:: [badeu](https://osu.ppy.sh/users/1473890) |

Reviewer: Name change. `night` to `G e n g a o z o`
```suggestion
| Playtester | ::{ flag=NL }:: [SecretlyShiro](https://osu.ppy.sh/users/7498203), ::{ flag=BE }:: [hexi](https://osu.ppy.sh/users/10760701), ::{ flag=BE }:: [Hanori](https://osu.ppy.sh/users/7078544), ::{ flag=NL }:: [Quinten](https://osu.ppy.sh/users/5101306), ::{ flag=FI }:: [mxu](https://osu.ppy.sh/users/18687807), ::{ flag=NL }:: [Lilily](https://osu.ppy.sh/users/6502403), ::{ flag=HK }:: [misha awa](https://osu.ppy.sh/users/14503423), ::{ flag=PT }:: [Kuronora](https://osu.ppy.sh/users/2742030), ::{ flag=NL }:: [faze-dank](https://osu.ppy.sh/users/13050074), ::{ flag=KR }:: [Amamya Kokoro](https://osu.ppy.sh/users/2511839), ::{ flag=NL }:: [Wittepoes](https://osu.ppy.sh/users/12984931), ::{ flag=NL }:: [UC2](https://osu.ppy.sh/users/6989615), ::{ flag=NL }:: [nik](https://osu.ppy.sh/users/10077264), ::{ flag=NL }:: [Alphan](https://osu.ppy.sh/users/13298387), ::{ flag=NL }:: [sofuraabu](https://osu.ppy.sh/users/7639453), ::{ flag=CN }:: [Crystal](https://osu.ppy.sh/users/1646397), ::{ flag=NO }:: [HUNDUR](https://osu.ppy.sh/users/3145033), ::{ flag=NL }:: [G e n g a o z o](https://osu.ppy.sh/users/14390731), ::{ flag=DE }:: [yary](https://osu.ppy.sh/users/13300203), ::{ flag=NO }:: [Melvr](https://osu.ppy.sh/users/9211924), ::{ flag=NL }:: [Kushper](https://osu.ppy.sh/users/4832514), ::{ flag=ES }:: [A L E P H](https://osu.ppy.sh/users/6735738), ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050), ::{ flag=RO }:: [badeu](https://osu.ppy.sh/users/1473890) |
```
